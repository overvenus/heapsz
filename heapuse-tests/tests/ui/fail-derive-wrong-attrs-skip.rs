use heapuse_derive::Heap;
use heapuse_example::*;

#[derive(Heap)]
pub struct WrongSkip1 {
    #[heap_size(skip)]
    pub a: primitive::StructPrimitives,
}

#[derive(Heap)]
pub enum WrongSkip2 {
    #[heap_size(skip)]
    A(usize),
    B(usize),
}

fn main() {}
