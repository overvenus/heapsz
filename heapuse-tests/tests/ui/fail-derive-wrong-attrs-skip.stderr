error: `#[heap_size(skip)]` is only allow with a container attribute `#[heap_size]`.
 --> tests/ui/fail-derive-wrong-attrs-skip.rs:6:17
  |
6 |     #[heap_size(skip)]
  |                 ^^^^

error: `#[heap_size(skip)]` is only allow with a container attribute `#[heap_size]`.
  --> tests/ui/fail-derive-wrong-attrs-skip.rs:12:17
   |
12 |     #[heap_size(skip)]
   |                 ^^^^
